jetty-server/src/main/java/org/eclipse/jetty/server/session/SessionHandler.java

```diff
@@ -725,7 +725,7 @@ public HttpCookie getSessionCookie(HttpSession session, String contextPath, bool
         if (isUsingCookies())
         {
             String sessionPath = (_cookieConfig.getPath()==null) ? contextPath : _cookieConfig.getPath();
-            sessionPath = (sessionPath==null||sessionPath.length()==0) ? "/" : sessionPath;
+            sessionPath = (sessionPath==null|| sessionPath.isEmpty()) ? "/" : sessionPath;
```

Comment:
might be a bit nitpicking but do not add space here

Reply:
As long as this is the only problem ...
No problem!
;-)
Done.